.github/workflows/promote.yaml

```diff
@@ -0,0 +1,197 @@
+name: Promote release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to promote (e.g.: 23.0.1-rc.2)'
+        required: true
+      referenceBranch:
+        description: 'Branch to be updated after the promotion'
+        required: true
+        default: 'dev'
+      dryRun:
+        description: 'Run the workflow without pushing code or publishing artifacts'
+        type: boolean
+        required: false
+      isMinorRelease:
+        description: "Is this a Enhancement Packs(EP's) or Service Packs(SP's) aka Minor Release (eg: 23.1.1 for 1st minor release after 23.1.0)"
+        type: boolean
+        required: false
+      isLastMinorRelease:
+        description: "Check the box if last release was minor release"
```

Comment:
Is this one needed to be input by user ? Can we not check the previous version ?